src/xcode/ENA/ENA/Source/Services/LocalStatistics/LocalStatisticsProviding.swift

```diff
@@ -17,7 +17,7 @@ protocol LocalStatisticsFetching {
 	typealias LocalStatisticsCompletionHandler = (Result<LocalStatisticsResponse, Error>) -> Void
 
 	func fetchLocalStatistics(
-		administrativeUnit: String,
+		groupID: String,
```

Comment:
```suggestion
		groupID: GroupIdentifier,
```
…or whatever is defined then. We have the `typealias` let's use it :)